### PR TITLE
update snapshot url in gbs.conf template

### PR DESCRIPTION
**Pull request by nn1a**
### Change Description ###

 - Update the snapshot URL in the gbs.conf template to the new URL.
 - remove usless buildconf

### CODE REVIEW

**Comment by jhchoilee** [comment]
Confirm

File: docs/platform/developing/setting-up.md [hunk 1]
```diff
@@ -210,13 +208,13 @@ Other examples:
 Each `profile` entry in the `.gbs.conf` file specifies multiple `repo` entries, and each `repo` entry specifies a URL where RPM files used in the GBS build are located.
 
 > [!NOTE]
-> The `latest` directory in the remote repository URLs is a symbolic link in the remote server, which is always linked to the latest new directory and can be changed any time, so make sure to use the latest repo with a specific date to guarantee usability. An example is shown below:
+> The `reference` directory in the remote repository URLs is a symbolic link in the remote server, which is always linked to the latest new directory and can be changed any time, so make sure to use the latest repo with a specific date to guarantee usability. An example is shown below:
```
Comment (omar-srbd) [inline docs/platform/developing/setting-up.md:211]:
```suggestion
> The `reference` directory in the remote repository URLs is a symbolic link in the remote server, which is always linked to the latest directory and can be changed any time, so make sure to use the latest repo with a specific date to guarantee usability. An example is shown below:
```

File: docs/platform/developing/setting-up.md [hunk 2]
```diff
@@ -210,13 +208,13 @@ Other examples:
...
 > ```
-> url = http://download.tizen.org/releases/daily/tizen/unified/latest/repos/standard/packages/
+> url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/reference/repos/standard/packages/
 > ```
-> This URL is symbolically linked to the latest snapshot number in "[http://download.tizen.org/releases/daily/tizen/unified/](http://download.tizen.org/releases/daily/tizen/unified/)". To guarantee usability, use a specific date:
+> This URL is symbolically linked to the latest snapshot number in "[http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/](http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/)". To guarantee usability, use a specific date:
```
Comment (omar-srbd) [inline docs/platform/developing/setting-up.md:215]:
```suggestion
> This URL is symbolically linked to the latest snapshot number in "[http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/](http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/){:target="_blank"}". To guarantee usability, use a specific date:
```

**Review by omar-srbd — COMMENTED** [review]
Hello @nn1a, please check the change suggestions and incorporate them.

File: docs/platform/developing/setting-up.md (hunk 1 above)
Comment (nn1a) [reply docs/platform/developing/setting-up.md]: done

File: docs/platform/developing/setting-up.md (hunk 2 above)
Comment (nn1a) [reply docs/platform/developing/setting-up.md]: done

**Review by omar-srbd — APPROVED** [review]
@nn1a, thanks for incorporating the changes.

@asm-foysal, approving this PR for merging.

**Pull request merged**